app/src/pages/metamine/visualizationNU/ScatterPlot.vue

```diff
@@ -1,20 +1,41 @@
 <template>
-    <div class="md-gutter adjust-padding wrapper-box">
-      <div class="main-content" style="margin: 3rem; display: flex">
-        <div class="scatter-chart" style="width: 50%">
-          <Scatter />
-        </div>
-        <div class="subcharts" style="width: 20%">
-          <Structure />
-          <Youngs />
-          <Poisson />
-        </div>
-        <div class="side-tools" style="width: 30%;">
-          <DataSelector />
-          <RangeSelector />
-          <MaterialInformation />
+  <div class="md-gutter adjust-padding wrapper-box">
+    <div class="main-content" style="margin: 3rem; display: flex">
+      <div class="scatter-chart" style="width: 50%">
+        <Scatter />
+        <div style="display:flex; justify-content: space-between; margin-top: 40px;">
+          <button @click="toggleDialogBoxKnn()" class="nuplot-button">Find Nearest Neighbors</button>
+        <dialog-box :active="dialogBoxActiveKnn">
+          <template v-slot:content>
```

Comment:
Just checking here whether it's intentional that the title is inside of the NeighborPanel content component rather than using `<template v-slot:title>`

Reply:
Yes it is intentional